Network/Network.cpp

```diff
@@ -917,7 +917,7 @@ namespace WPEFramework
                 if (!autoconfig)
                 {
                     RFC_ParamData_t param;
-                    if (Utils::getRFCConfig("Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.Network.ManualIPSettings.Enable", param))
+                    if (Utils::getRFCConfig(const_cast<char*>("Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.Network.ManualIPSettings.Enable"), param))
```

Comment:
Instead of casting away the constness, change the function to take a constant parameter

./helpers/utils.h:    bool **getRFCConfig(**const** char* paramName, RFC_ParamData_t& paramOutput);

Reply:
Agree and I'll do it.